examples/pytorch_lightning_track.py

```diff
@@ -103,7 +103,7 @@ def cli_main():
     # ------------
     # testing
     # ------------
-    trainer.test(test_dataloaders=test_loader)
```

Comment:
is this related to the fix?

Reply:
nope, just found out during the testing, and fixed it :)